website/content/docs/overview/modules/index.md

```diff
@@ -6,120 +6,60 @@ weight: 40
 
-At runtime, the control plane builds a blueprint for each application. The blueprint includes additional components that are injected into the data path by the control plane to fulfill governance requirements and application requirements.
+The functionality described by the module may be deployed (a) per workload, or (b) it may be a component that runs independent of the workload and its associated control plane.  In the case of (a), the Mesh for Data control plane handles the deployment of the functional component. In the case of (b) where the functionality of the module runs independently and handles requests from multiple workloads, a client module is what is deployed by the Mesh for Data control plane.  This client module passes parameters to the external component and monitors that status and results of the request to the external component.  The external component is declared as a dependency in the module yaml.
 
-Modules are the way to provide a specification of the capabilities of such components, and make them available to the control plane.
+In both cases, the module component is packaged as a [Helm](https://helm.sh/) chart that the control plane can install to a workload's data plane. To make a module available to the control plane it must be registered by applying a [`M4DModule`]({{< baseurl >}}/docs/reference/api/generated/app/#k8s-api-github-com-ibm-the-mesh-for-data-manager-apis-app-v1alpha1-m4dmodule) CRD.
 
-# Concept
+The following diagram shows an example with an Arrow Flight module that is fully deployed by the control plane and a second module where the client is deployed by the control plane but the ETL component providing the functionality has been independently deployed and supports multiple workloads.
 
-For the control plane to build a blueprint that includes modules, the control plane needs a catalog of all the modules available for its use.  Therefore, the control plane includes a **module repository** listing all registered modules. 
+![Example](ModuleArch2.png)
 
 
-For each module, the control plane needs **module details** describing the capabilities of the module (read, write, copy, transforms, etc.). This is needed so the control plane can pick the set of modules that can be used together in the same blueprint and fulfill all application and governance requirements. 
+# Components that make up a module
+There are several parts to a module:
+1. [Module Workload](#module-workload): the workload that runs once the Helm chart is installed by the control plane
+2. (Optional) External component: deployed and managed independently of the Mesh for Data.  In this case, the [Module Workload](#module-workload) is a client to the external component.
```

Comment:
Same comment as above, use case specific. I think that we can simplify yet keep this specific use case as an example pattern.

Reply:
Updated